PayPal/src/main/java/com/braintreepayments/api/PayPalCheckoutRequest.java

```diff
@@ -207,7 +207,9 @@ String createRequestBody(Configuration configuration, Authorization authorizatio
             parameters.put(BILLING_AGREEMENT_DETAILS_KEY, details);
         }
 
-        parameters.putOpt(PAYER_EMAIL_KEY, userAuthenticationEmail);
+        if (userAuthenticationEmail != null && !userAuthenticationEmail.isEmpty()) {
+            parameters.putOpt(PAYER_EMAIL_KEY, userAuthenticationEmail);
```

Comment:
You could just do `put` instead of `putOpt` at this point.

Reply:
Good catch. let me update this.